src/UI/home/dashboard/types/THistoryItem.tsx

```diff
@@ -0,0 +1,10 @@
+export type THistoryItem = {
+  id: number
+  type: string
+  amount: number
+  coin: string
+  price: number
+  comment: string
+  date: string
+  key: number
+}
```

Comment:
awesome work with creating these types :+1: it may be frustrating at first but believe me, when the project gets bigger it solves a lot of problems